contracts/test-utils/src/order_factory.ts

```diff
@@ -19,12 +19,12 @@ export class OrderFactory {
         customOrderParams: Partial<Order> = {},
         signatureType: SignatureType = SignatureType.EthSign,
     ): Promise<SignedOrder> {
-        const tenMinutesInSeconds = 10 * 60;
+        const fifteenMinutesInSeconds = 15 * 60;
```

Comment:
`contracts/exchange/test/match_orders.ts` was taking too long and some orders were expired before the test suite finished.